[rospy.client][INFO] 2020-11-12 03:27:19,906: init_node, name[/train_9073_1605151639901], pid[9073]
[xmlrpc][INFO] 2020-11-12 03:27:19,907: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-12 03:27:19,907: Started XML-RPC server [http://luigi:38335/]
[rospy.init][INFO] 2020-11-12 03:27:19,908: ROS Slave URI: [http://luigi:38335/]
[rospy.impl.masterslave][INFO] 2020-11-12 03:27:19,908: _ready: http://luigi:38335/
[rospy.registration][INFO] 2020-11-12 03:27:19,909: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-12 03:27:19,909: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-12 03:27:20,008: registered with master
[rospy.rosout][INFO] 2020-11-12 03:27:20,008: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-12 03:27:20,014: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-12 03:27:20,015: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-12 03:27:20,035: topic[/bev/flow_image] adding connection to [http://192.168.0.201:37255/], count 0
[rospy.internal][INFO] 2020-11-12 03:27:20,038: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:37255/], count 0
[rospy.internal][INFO] 2020-11-12 03:27:20,040: topic[/odom] adding connection to [http://192.168.0.201:46833/], count 0
[rospy.internal][INFO] 2020-11-12 03:27:20,281: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-11-12 03:27:59,519: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-11-12 03:27:59,524: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-12 03:27:59,525: topic[/bev/flow_image] removing connection to http://192.168.0.201:37255/
[rospy.internal][INFO] 2020-11-12 03:27:59,525: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:37255/
[rospy.internal][INFO] 2020-11-12 03:27:59,526: topic[/odom] removing connection to http://192.168.0.201:46833/
[rospy.impl.masterslave][INFO] 2020-11-12 03:27:59,526: signal-2
[rospy.internal][WARNING] 2020-11-12 03:27:59,527: Unknown error initiating TCP/IP socket to 192.168.0.201:56173 (http://192.168.0.201:37255/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2020-11-12 03:28:00,019: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 03:28:00,061: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 03:28:00,081: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 03:28:00,111: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 03:28:00,127: signal_shutdown [atexit]
